doc/classes/bool.xml

```diff
@@ -6,24 +6,50 @@
 	<description>
 		Boolean is a built-in type. There are two boolean values: [code]true[/code] and [code]false[/code]. You can think of it as an switch with on or off (1 or 0) setting. Booleans are used in programming for logic in condition statements, like [code]if[/code] statements.
 		Booleans can be directly used in [code]if[/code] statements. The code below demonstrates this on the [code]if can_shoot:[/code] line. You don't need to use [code]== true[/code], you only need [code]if can_shoot:[/code]. Similarly, use [code]if not can_shoot:[/code] rather than [code]== false[/code].
-		[codeblock]
+		[codeblocks]
+		[gdscript]
 		var can_shoot = true
 
 		func shoot():
 		    if can_shoot:
 		        pass # Perform shooting actions here.
-		[/codeblock]
+		[/gdscript]
+		[csharp]
+        private bool _canShoot = true;
```

Comment:
The code blocks should start with tabs in the XML documentation. There are several more problems like this.

Reply:
fixed. That was the one class, I converted by hand, when I implemented the feature first-hand ^^

Reply:
yeah that was the one class I converted by hand to test the feature when I implemented it...

Reply:
fixed